tests/integrationv2/processes.py

```diff
@@ -172,9 +177,13 @@ def _communicate(self, input_data=None, timeout=None):
                         # If we are looking for, and find, the ready-to-send marker, then
```

Comment:
nit: `ready-to-send`/ `send_marker`

Reply:
😄 My search function must have missed this one, thanks!